middleware/components/video-placeholder.js

```diff
@@ -0,0 +1,157 @@
+const { Component } = wp.element;
+const { __ } = wp.i18n;
+
+const {
+	RichText,
+	MediaUpload,
+	BlockControls,
+} = wp.blocks;
+
+const {
+	Placeholder,
+	FormFileUpload,
+	Button,
+	Toolbar,
+	IconButton,
+} = wp.components;
+
+const { mediaUpload } = wp.utils;
+
+class VideoPlaceholder extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.state = {
+			editing: ! this.props.videoData,
+			src: this.props.videoData || '',
```

Comment:
Shouldn't this be an object instead of string and name should not be `src` ?

Reply:
It will be object only. Also, I have replace variable name, Please check.